### PR TITLE
Adds first inference time measurements in benchmark_app

### PR DESCRIPTION
**Pull request by nkogteva**

### CODE REVIEW

**Comment by nkogteva** [comment]
cc @ptelkov

File: inference-engine/samples/benchmark_app/main.cpp [hunk 1]
```diff
@@ -513,13 +513,19 @@ int main(int argc, char *argv[]) {
...
             THROW_IE_EXCEPTION << "No idle Infer Requests!";
         }
-
         if (FLAGS_api == "sync") {
             inferRequest->infer();
         } else {
             inferRequest->startAsync();
         }
         inferRequestsQueue.waitAll();
+        auto duration_ms = double_to_string(inferRequestsQueue.getLatencies()[0]);
```
Comment (ilya-lavrenov) [inline inference-engine/samples/benchmark_app/main.cpp:522]: does it return read + load + single inference?

Comment (nkogteva) [reply inference-engine/samples/benchmark_app/main.cpp:522]: We report read, load and reshape times and time of first inference.  Full "first inference time" actually depends on real scenario. These 3 (Read, Load, Infer) or 4 (+ Reshape) times can be easily summed up. But I think we should not mislead users doing it yourself.

Comment (ilya-lavrenov) [reply inference-engine/samples/benchmark_app/main.cpp:522]: ok, I see

**Pull request merged**